com.unity.multiplayer.mlapi/Runtime/Profiling/PerformanceDataManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace MLAPI.Profiling
+{
+    public static class PerformanceDataManager
```

Comment:
Should this class be internal? Do we ever expect external (non-MLAPI) code to call these APIs?

If we do, I think we should at least make `BeginNewTick`, `AddTransportData`, and `GetData` to be internal at the member level